src/App.tsx

```diff
@@ -1,24 +1,216 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
 
-const USER_ID = 0;
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import { Todo } from './types/Todo';
+import { client } from './utils/fetchClient';
+import { TodoList } from './Components/TodoList/TodoList';
+import { FilterBy } from './utils/FilterBy';
+import { TodoFilter } from './Components/TodoFilter/TodoFilter';
+import { TodoItem } from './Components/Todo/TodoItem';
+
+const USER_ID = 10341;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [tempTodo, setTempTodo] = useState<Todo | null>(null);
+  const [filterBy, setFilterBy] = useState<FilterBy>(FilterBy.All);
+  const [query, setQuery] = useState<string>('');
+  const [isTitleEmpty, setIsTitleEmpty] = useState<boolean>(false);
+  const [isAddError, setIsAddError] = useState<boolean>(false);
+  const [isRemoveError, setIsRemoveError] = useState<boolean>(false);
+  const [isRemCompleted, setIsRemCompleted] = useState<boolean>(false);
+
+  const visibleTodos = useMemo(() => (
+    todos.filter((todo) => {
+      switch (filterBy) {
+        case FilterBy.Active:
+          return !todo.completed;
+
+        case FilterBy.Completed:
+          return todo.completed;
+
+        default:
+          return true;
+      }
+    })
+  ), [todos, filterBy]);
+
+  const getTodos = async () => {
+    const todosFromServer = await client.get<Todo[]>(`/todos?userId=${USER_ID}`);
```

Comment:
move it to the function
```suggestion
    const todosFromServer = await get....;
```